composer.json

```diff
@@ -47,7 +47,8 @@
         "twig/twig": "~1.28|~2.0",
         "doctrine/dbal": "~2.2",
         "swiftmailer/swiftmailer": "~5",
-        "monolog/monolog": "^1.4.1"
+        "monolog/monolog": "^1.4.1",
+        "phpunit/phpunit": "~5.7"
```

Comment:
I'm not super familiar with defining requirements for different versions of PHP. 

Should `"phpunit/phpunit": "~4.8|~5.7"` fix this?